src/style.css

```diff
@@ -0,0 +1,63 @@
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+.search-form {
+  margin: 20px 0;
+}
+
+.search-input {
+  width: 100%;
+  height: 70px;
+  color: #48485e;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 15px 24px;
+  background-size: 24px;
+  border: solid 0.5px #d3d3d3;
+  border-radius: 2px;
+  padding-left: 55px;
+  font-family: "Avenir-Heavy", Arial, sans-serif;
+  font-size: 26px;
```

Comment:
add transition for box-shadow here

Reply:
also add default box-shadow to the input (check the design)

Reply:
Could you check my task again please? I made changes two days ago.  Thanks